conda/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set data= load_setup_py_data() %}
 package:
-  name: ts-mteec
+  name: "ts-mteec"
```

Comment:
The double quotes are not needed (this is yaml). I suggest reverting this change

Reply:
Done